src/main/java/org/ladocuploader/app/submission/actions/SetExperimentGroups.java

```diff
@@ -0,0 +1,27 @@
+package org.ladocuploader.app.submission.actions;
+
+import formflow.library.config.submission.Action;
+import formflow.library.data.Submission;
+import java.util.Random;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SetExperimentGroups implements Action {
+  
+  public enum ExperimentGroup {
+    CONTROL,
+    LINK,
+    APPLY
+  }
+  
+  @Override
+  public void run(Submission submission) {
+    Random rand = new Random();
+    int group = rand.nextInt(3);
+    switch(group) {
+      case 0 -> submission.getInputData().put("experimentGroup", ExperimentGroup.CONTROL);
+      case 1 -> submission.getInputData().put("experimentGroup", ExperimentGroup.LINK);
+      case 2 -> submission.getInputData().put("experimentGroup", ExperimentGroup.APPLY);
```

Comment:
Do enums get saved as strings in the db? If there's some conversion needed, might be nice to have a helper method in the enum class

Reply:
Looks like they are automatically converted to strings. Here is how they look in the JSON:
<img width="192" alt="Screenshot 2023-12-14 at 10 39 45 AM" src="https://github.com/codeforamerica/la-doc-uploader/assets/39173486/a2e5bff8-3443-42e7-8745-b989486e982b">